### PR TITLE
[Auth] Login User

### PR DESCRIPTION
**Pull request by Nickoliz**

### CODE REVIEW

File: tacktivity/client/src/components/Navbar.js [hunk 1]
```diff
@@ -0,0 +1,9 @@
+// import React from 'react';
+
+// function Navbar() {
+//   const [ login, setLogin ] = setState('');
+
+
+// }
+
+// export default Navbar;
```
Comment (ssoonmi) [inline tacktivity/client/src/components/Navbar.js:9]: This file should be deleted

Comment (ssoonmi) [inline tacktivity/client/src/components/Navbar.js:9]: This file should be deleted

File: tacktivity/client/src/css/login.css [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+  display: block;
+}
+
+.signup-hidden {
+  display: none;
+}
+
+.signup-show {
+  display: block;
+} */
```
Comment (ssoonmi) [inline tacktivity/client/src/css/login.css:108]: Should remove this file

File: tacktivity/db/models/index.js [hunk 3]
```diff
@@ -12,7 +12,7 @@ let sequelize;
 if (config.use_env_variable) {
   sequelize = new Sequelize(process.env[config.use_env_variable], config);
 } else {
-  sequelize = new Sequelize(config.database, config.username, config.password, config);
+  sequelize = new Sequelize(config.database, config.email, config.password, config);
```
Comment (ssoonmi) [inline tacktivity/db/models/index.js:15]: the `username` here is for accessing the `database`, not the `Users` table's `username`

Comment (Nickoliz) [reply tacktivity/db/models/index.js:15]: This was resolved.

File: tacktivity/client/src/components/Navbar.js (hunk 1 above)
Comment (Nickoliz) [reply tacktivity/client/src/components/Navbar.js:9]: Removed.

**Comment by Nickoliz** [comment]
QQ on my splash page. The background is a placeholder since I'll need actual files to render the background - is it okay to  leave it as is until I get photos? I'm hoping that will be a Thursday task at this point since I'll need it for everything really.

**Comment by ssoonmi** [comment]
> QQ on my splash page. The background is a placeholder since I'll need actual files to render the background - is it okay to leave it as is until I get photos? I'm hoping that will be a Thursday task at this point since I'll need it for everything really.

Yes! that's fine

**Comment by Nickoliz** [comment]
Okay - struggling with the link to Sign up and Log in from within the modal. I think I'll have to create new routes to those like /login and /signup. That way if within the modal you click it you are taken to that route.  I think I'll first try to create a separate Modal component that I can place in my Unauthed component that will allow the toggle between Login and Signup components.

**Comment by ssoonmi** [comment]
> Okay - struggling with the link to Sign up and Log in from within the modal. I think I'll have to create new routes to those like /login and /signup. That way if within the modal you click it you are taken to that route. I think I'll first try to create a separate Modal component that I can place in my Unauthed component that will allow the toggle between Login and Signup components.

You can pass in the function to toggle the modal from login to signup, etc. into the Login and Signup components.

File: tacktivity/routes/api/session.js [hunk 4]
```diff
@@ -12,8 +12,8 @@ const { jwtConfig: { expiresIn } } = require('../../config');
 const router = express.Router();
 
 const validateLogin = [
-  check("email").exists(),
-  check("password").exists(),
+  check("email", "Oi! You missed a spot! Gonna need your email.").exists(),
+  check("password", "What's the password?").exists(),
```
Comment (ssoonmi) [inline tacktivity/routes/api/session.js:16]: 😄  💎

**Comment by Nickoliz** [comment]
Background image resolved.

**Pull request merged**